k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1208,14 +1215,6 @@ function deployCortxClient()
     printf "\n\n"
 }
 
-function deployCortxServices()
```

Comment:
Do you need to remove the corresponding function deleteCortxServices() in destroy-cortx-cloud.sh?